python/pyspark/sql/functions.py

```diff
@@ -1300,12 +1300,19 @@ def to_utc_timestamp(timestamp, tz):
     zone, and renders that time as a timestamp in UTC. For example, 'GMT+1' would yield
     '2017-07-14 01:40:00.0'.
```

Comment:
I have thought about this for long time and let's start to describe what we change within Python side:

```
        :param timestamp: the column that contains timestamps
        :param tz: a string that has the ID of timezone, e.g. "GMT", "America/Los_Angeles", etc.

        .. versionchanged:: 2.4
           `tz` can take a :class:`Column` containing timezone ID strings.
```

Reply:
ok